### PR TITLE
Text changes april2023 onwards

### PR DESCRIPTION
**Pull request by manishjain999**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -1262,14 +1278,14 @@
...
-                    "Edit activities for your morning routine, evening routine, breaks throughout the day.",
+                    "Edit activities for your morning routine, evening routine, breaks throught the day.",
                     "Setup or change Password.",
                     "View your progress stats."
                 ]
             },
             {
                 "title": "Setup Focus mode",
-                "description": "Setup focus modes to get into deep work with distraction blocking. You can remove blocks if you have an emergency.",
+                "description": "Setup focus modes to et into deep work with distraction blocking. You can remove blocks if you have an emergency.",
```
Comment (jeznag) [inline config/config.json:1288]: revert

File: config/config.json [hunk 2]
```diff
@@ -1326,7 +1344,19 @@
...
+    "timeTrackerVcInfo": {
+        "title": "Introducing Focus Bear Time Tracker",
+        "description": "Focus Bear Time Tracker helps you analyse your productivity by tracking how much time you spend on different websites and apps. You can automatically categorise time logs into different projects (helpful for timesheets).",
+        "pointFirstText": "Privacy first: data is stored locally on your Mac",
+        "pointSecondText": "Assign time logs to different projects (helpful for timesheets)",
+        "pointThirdText": "Discover where you're wasting time (you can block it with Focus Bear 😉)",
+        "pointFourthText": "Track time on meetings and when you're away from your Mac",
+        "buttonEnableText": "Enable Focus Bear Time Tracker",
+        "buttonCancelText": "No thanks",
+        "alertInfoText": "You can always enable Focus Bear Time Tracker from Focus Bear prefrences later."
```
Comment (jeznag) [inline config/config.json:1359]: typo preferences

File: config/config.json [hunk 3]
```diff
@@ -1262,14 +1278,14 @@
                 "title": "Preferences",
                 "description": "Click on \"Preferences\" to",
                 "arrayListItems": [
-                    "Edit activities for your morning routine, evening routine, breaks throughout the day.",
+                    "Edit activities for your morning routine, evening routine, breaks throught the day.",
```
Comment (jeznag) [inline config/config.json:1281]: revert

Comment (manishjain999) [reply config/config.json:1281]: done

File: config/config.json (hunk 1 above)
Comment (manishjain999) [reply config/config.json:1288]: Done

File: config/config.json (hunk 2 above)
Comment (manishjain999) [reply config/config.json:1359]: fixed

**Pull request merged**